debian/postinst

```diff
@@ -36,6 +36,10 @@ case "$1" in
       adduser --no-create-home --group --system ${USER}
     fi
     set -e
+    touch /var/log/${PACKAGE}
```

Comment:
All references to `/var/log`, `/var/lib`, and `/etc` should be eliminated.  None of those directories are used by the software by default or explicitly by the documentation.  Instead, that information is all kept in `~/.local/share/eosio`.  The old packages needed to install a `genesis.json` file, which is also no longer required.

With the elimination of installation to both `/var` and `/etc`, it's likely that usage of `CMAKE_INSTALL_FULL_*` is no longer required for any component.

Reply:
Not sure I follow what you expect.
```
message(WARNING ${CMAKE_INSTALL_FULL_BINDIR})
message(WARNING ${CMAKE_INSTALL_BINDIR})
```

This gives
```
CMake Warning at CMakeLists.txt:12 (message):
  /usr/local/bin
CMake Warning at CMakeLists.txt:13 (message):
  bin
```